src/backend/app.js

```diff
@@ -64,6 +64,43 @@ app.post('/api/auth/login', function (req, res) {
   });
 });
 
+app.post('/api/auth/token', function (req, res) {
+
+  if (req.body.email && req.body.password) {
+    var email = req.body.email;
+    var password = req.body.password;
+    var user = users.find(function (u) {
+      return u.email === email && u.password === password;
+    });
+    if (user) {
+      var payload = {
+        id: user.id,
+        email: user.email,
+        role: 'user',
+      };
+      var refreshPayload = {
```

Comment:
why do we need an additional payload for a refresh token?

Reply:
I don't understand what you mean.
In the response we have to send an access_token and a refresh_token so i need 2 payloads for 2 tokens?
What do you think ?

Reply:
You are right, the specification does not impose anything about the refresh token.
I have seen some implementations of refresh_token considering it can be a JWT refreshToken
But it does not have to.
So you're right, i will fall back to basic string.